misc/userscripts/iframe

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import os
+import sys
+import bs4
+import re
+
+
+class StaticGetAttrType(type):
+    """Metaclass which simply allows class to define
+    a __getstaticattr__ method.
+    __getstaticattr__ is the static equivalent of __getattr__.
+    """
+    def __getattr__(cls, name):
+        return cls.__getstaticattr__(name)
+
+
+class Env(metaclass=StaticGetAttrType):
+    """Helper class which allows to access environment variables"""
+    def __getstaticattr__(name):
+        return os.environ[name]
```

Comment:
This seems to be a lot of magic just to avoid writing `os.environ` below. Really, just use `os.environ['QUTE_FIFO']` etc. instead :wink: